vach/src/lib.rs

```diff
@@ -15,7 +15,7 @@ pub(crate) mod writer;
 pub use rand;
 
 /// Current file spec version, both `Loader` and `Builder`
-pub const VERSION: u16 = 13;
+pub const VERSION: u16 = 15;
```

Comment:
if the crate's version is now 0.1.6, shouldn't this be 16?

Reply:
Oh yeah!